app/services/user.js

```diff
@@ -0,0 +1,49 @@
+const bcrypt = require('bcryptjs');
+
+const db = require('../models');
+const errors = require('../errors');
+const userConfig = require('../../config').common.user;
+const saltRounds = 10;
+
+exports.checkUserProperties = query => {
+  if (query.firstName && query.lastName && query.email && query.password) {
```

Comment:
you cannot use this condition for return directly?

return (query.firstName && query.lastName && query.email && query.password)